.eslintrc.js

```diff
@@ -16,6 +16,7 @@ module.exports = {
   ],
   rules: {
     'mocha/handle-done-callback': 'off',
+    '@typescript-eslint/no-unused-vars': 'off',
```

Comment:
@mrmodise I also noticed this issue on my applications when updating dev dependencies, I don't think adding this rule is a good solution

Reply:
ah! I guess we can wait updating @raymondfeng @bajtos your thoughts

Reply:
Let's revert this change and find a way how to fix the problematic code instead. The easiest solution is to disable `@typescript-eslint/no-unused-vars` only for the problematic lines. It would be better to take a look at how we are dealing with imported generic types in loopback-next. IIRC, we have encountered this problem there too.

In Dec 2020, we upgraded typescript-eslint monorepo to ^4.9.0 and fixed violations this way:
- https://github.com/strongloop/loopback-next/pull/6863/commits/007ee27a7fcfe3b446f51c198fa79fc687e098fb

In Jan 2021, we upgraded typescript-eslint monorepo to ^4.14.0 which allowed us to remove most (if not all) comments added for v4.9.0, see:
- https://github.com/strongloop/loopback-next/pull/6888/commits/1729ad272bb5f94973c617ce61226c8e6ee60f3c

So maybe we need to upgrade typescript-eslint to latest and only then continue with other changes? If we are consuming typescript-eslint via `@loopback/eslint-config`, then we should start with letting RenovateBot to update this pull request to fetch the latest LB4 release, in particular `@loopback/eslint-config@10.0.5`. Let me try to do that right now.

Reply:
The change is already reverted and the pipeline is passing @bajtos Thanks for checking this out